scripts/KeyVaultWrapper.py

```diff
@@ -1,15 +1,19 @@
 from azure.keyvault import KeyVaultClient
 from azure.common.credentials import ServicePrincipalCredentials
+from msrestazure.azure_active_directory import MSIAuthentication

```

Comment:
You should do a pip freeze > requirements.txt. It will make it easier to install all of the necessary deps

Reply:
Could be. msrestazure is installed as a azure-keyvault dependency now.. We can make it an explicit dependency.